src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -2798,7 +2803,7 @@ private void ClassifyUpdate(AttributeListSyntax oldNode, AttributeListSyntax new
                 // changes in attribute separators are not interesting:
             }
 
-            private static bool AreModifiersEquivalent(SyntaxTokenList oldModifiers, SyntaxTokenList newModifiers, SyntaxKind ignore)
+            private static bool AreModifiersEquivalent(SyntaxTokenList oldModifiers, SyntaxTokenList newModifiers, SyntaxKind ignore, SyntaxKind? ignore2 = null)
```

Comment:
Let me know if you don't prefer the way I did it here.

Reply:
I'd probably prefer it if the IndexOf and RemoveAt were just part of a local function that was called twice, but I don't feel too strongly about it.